tests/python/pants_test/reporting/test_reporting_integration.py

```diff
@@ -177,7 +179,7 @@ def test_zipkin_reporter(self):
       num_of_traces = len(ZipkinHandler.traces)
       self.assertEqual(num_of_traces, 1)
 
-      trace = ZipkinHandler.traces[-1]
+      trace = next(iter(ZipkinHandler.traces.values()))
```

Comment:
Rather than `next(iter(` maybe call `assert_single_element` from `pants.util.collections`? (And throughout)